### PR TITLE
DOC-8232: Docs for Datadog metrics integration supported for Dedicated Azure clusters

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses: DOC-8232

(1) Updated Observability section of CockroachDB Dedicated on Azure page. (2) Updated Export Metrics From a CockroachDB Dedicated Cluster page, added “or Azure” to the four instances that “GCP” was mentioned.

### CODE REVIEW

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 1]
```diff
@@ -36,8 +36,9 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
 ### Observability
 
+- Exporting metrics to [Datadog](https://www.datadoghq.com/) is available. Enable the Datadog integration in [your {{ site.data.products.dedicated }} Cloud Console]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-dedicated-with-datadog), or on the [command line via the Cloud API]({% link cockroachcloud/export-metrics.md %}?filters=datadog-metrics-export).
```
Comment (gemma-shay) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:39]:
```suggestion
- Exporting metrics to [Datadog](https://www.datadoghq.com/) is available. Enable the Datadog integration in the [CockroachDB {{ site.data.products.cloud }} Console]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-dedicated-with-datadog) or with the [Cloud API]({% link cockroachcloud/export-metrics.md %}?filters=datadog-metrics-export).
```

**Review by gemma-shay — APPROVED** [review]
One comment but lgtm otherwise!

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md (hunk 1 above)
Comment (florence-crl) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md]: updated

**Review by kevin-v-ngo — APPROVED** [review]
Nice!

**Pull request merged**